frontend/packages/kubevirt-plugin/integration-tests/tests/vm.guest.agent.scenario.ts

```diff
@@ -31,67 +30,77 @@ import {
   VM_CREATE_AND_EDIT_AND_CLOUDINIT_TIMEOUT_SECS,
   VM_WITH_GA_CREATE_AND_EDIT_CLOUDINIT_TIMEOUT_SECS,
   GUEST_AGENT_FIELD_TIMEOUT_SECS,
+  SEC,
+  KUBEVIRT_SCRIPTS_PATH,
 } from './utils/constants/common';
 import { VirtualMachine } from './models/virtualMachine';
 import { getFakeWindowsVM } from './utils/templates/windowsVMForRDPL2';
-import { multusNAD } from './mocks/mocks';
-
-const VM_NAME = 'windows-rdp';
-const VM_IP = '123.123.123.123';
+import { getVMManifest, multusNAD } from './mocks/mocks';
+import { ProvisionSource } from './utils/constants/enums/provisionSource';
+import { VM_STATUS, VM_ACTION } from './utils/constants/vm';
 
+const VM_LINUX_NAME = `${testName}-linux-vm`;
+const VM_WINDOWS_NAME = 'windows-rdp';
+const VM_WINDOWS_IP = '123.123.123.123';
 describe('Tests involving guest agent', () => {
-  let vm: VirtualMachine;
-
+  let vm_linux: VirtualMachine;
+  let vm_windows: VirtualMachine;
+  
   beforeAll(async () => {
     createResource(multusNAD);
 
-    // for cmd-line scripts only
+    //create linux vm
+      const cloudInit = `#cloud-config\nuser: cloud-user\npassword: atomic\nchpasswd: {expire: False}\nruncmd:\n- dnf install -y qemu-guest-agent\n- systemctl start qemu-guest-agent`;
+      const testVM = getVMManifest(ProvisionSource.CONTAINER, testName, VM_LINUX_NAME, cloudInit);
+      vm_linux = new VirtualMachine(testVM.metadata);
+      createResource(testVM);
+      await vm_linux.detailViewAction(VM_ACTION.Start, false);
+      await vm_linux.waitForStatus(VM_STATUS.Running, SEC * 600);
+      await browser.sleep(GUEST_AGENT_FIELD_TIMEOUT_SECS);
+      await vm_linux.detailViewAction(VM_ACTION.Restart, false);
```

Comment:
Restart is not necessary.

Reply:
restart is necessary (checked more than once)

Reply:
Could you explain why it needs to restart here? The VM is provisioned from container, nothing is reserved after restart the VM outside.

Reply:
the guest agent started only after restart

Reply:
not at all, guest agent is started very soon after run `systemctl start qemu-guest-agent` in cloud-init, it might need to wait a one minutes or two to see it's running. Restart is not necessary and it does not help anything.